CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/hierarchical/HierarchicalControllerTopComponent.java

```diff
@@ -551,7 +560,14 @@ private void setGroups(final boolean doUpdate) {
             if (state != null && doUpdate) {
                 updateGraph();
             }
+            if(state != null && state.interactive) {
+                interactiveButton.setText(INTERACTIVE_ENABLED);
+            } else {
+                interactiveButton.setText(INTERACTIVE_DISABLED);
+            }
```

Comment:
Up to you but given the if and else code are almost exactly the same bar the parameter value, can changed this to:
`interactiveButton.setText((state != null && state.interactive) ? INTERACTIVE_ENABLED : INTERACTIVE_DISABLED)`

Reply:
I'm all for changing it - I originally had it as a nested ternary and SQ found a code smell. I'll change it and see how SQ likes it